packages/components/charts/tests/area-chart.test.tsx

```diff
@@ -671,7 +671,7 @@ describe("<AreaChart />", () => {
     await expect(screen.findByText(/x line/i)).resolves.toBeInTheDocument()
   })
 
-  test("render valueFormatter", async () => {
+  test("should be renderd valueFormatter", async () => {
```

Comment:
It looks like there's a typo in the test description. "renderd" should be corrected to "rendered".

Reply:
@kurakke 
Fixed.